gratipay/models/participant.py

```diff
@@ -416,11 +417,11 @@ def add_email(self, email, resend_threshold='3 minutes'):
             nonce = self.db.one("""
                 UPDATE emails
                    SET verification_start=%s
-                 WHERE participant=%s
+                 WHERE participant_id=%s
                    AND address=%s
                    AND verified IS NULL
              RETURNING nonce
-            """, (verification_start, self.username, email))
+            """, (verification_start, self.id, email))
             if not nonce:
                 return self.add_email(email)
 
```

Comment:
ok I have no idea what the problem was but I couldn't even run the site locally at `localhost:8537`. `make run` kept exiting. So I restarted my PC and now everything works! All tests pass. :dancer: